src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -110,6 +123,30 @@ export class MainThreadConnectionManagement implements MainThreadConnectionManag
 		return connection;
 	}
 
+	private convertToConnectionProfile(profile: IConnectionProfile): azdata.connection.ConnectionProfile {
```

Comment:
>convertToConnectionProfile [](start = 9, length = 26)

This feels like it should be a method on ConnectionProfile instead

Reply:
ConnectionProfile is an internal class, there I am converting it to the public azdata.connection.ConnectionProfile